### ISSUE TITLE
Add "Clear Cache" functionality for a better ux

### ISSUE THREAD

**Issue by 00travelgirl00** [pluginkollektiv/cachify#151]
While using cachify I'm missing the "clear cache" functionality, because for unexperienced users it is not quiet clear how to clear the caching and to check their changes. 

I have created a patch and a mockup, which only includes the design, not the functionality! 

PR: #150 

<img width="1232" alt="bildschirmfoto 2018-02-05 um 22 58 52" src="https://user-images.githubusercontent.com/21318377/35831374-b26beb42-0ac9-11e8-9ce8-3fa586c3d805.png">

**Comment by te-online** [comment]
While there is the trash can in the admin bar to empty the cache, the icon is neither very descriptive nor does it seem available for non-admin users.

I'd rather add a label to the trash-can and/or replace the icon in the admin bar and additionally make it configurable to make it available to non-admin users as well.

**Comment by 00travelgirl00** [comment]
I not even noticed the trash can and did not know, its relate to the caching plugin. I think its important that you have the option to clear the cache also in the settings page of cachify, because thats the place where new users would look at. 

Maybe we can add the trash can symbol next to expiration with the label "clear cache" and the description "you can also use the trash can symbol in the admin bar". 

In addition we could also label the trash can in the admin bar, to be more clear.

**Comment by te-online** [comment]
Yeah, it's a huge usability problem. I think your suggestions are good, and I'll see if I can get a pull-request done in the upcoming weeks.

**Comment by websupporter** [comment]
@te-online,
thanks for having a look into this, please keep also the discussion on the PR #150 in mind, when drilling into this problem. :heart:

**Comment by Zodiac1978** [comment]
Related: https://gist.github.com/glueckpress/0ccc84e6f93b89dee1d3

**Comment by dionysous** [comment]
PR #150 is now merged. I relocated the flush button right under the input field in the **cache expiration** section. The buttons is colored orange.

When I used to visit the Cachify plugin settings page, I was always looking for a **flush button**. That's why I worked on @00travelgirl00 PR.

<img width="1018" alt="screenshot: cachify settings page with orange flush button" src="https://user-images.githubusercontent.com/12687777/45818931-656c6200-bce3-11e8-965c-000156d3de60.png">

**Comment by te-online** [comment]
@dionysous This looks awesome! 😊  (I haven't had the time to do anything regarding the caption in the admin bar, though... 😳 )

**Comment by te-online** [comment]
@dionysous In addition to adding a description text, which sounds like a really good idea, I'd really prefer to add a label to the “trashcan”. 

My idea is to have maybe have a checkbox on the settings page or something (yet, another checkbox... ;-)) to enable the label, in order to not disturb what it looks like for people used to the trashcan only, who know what this button does. But, the reason why I'm suggesting to do this is: There has literally not been *one single user*, in the years I've handed over WordPress sites to users, who understood that this button was to flush the cache without me explaining it to them. That's like having an odd button at one side of the toilet to flush it, not really obvious for most people 😉 

Saying that, I'm certainly open for alternative solutions to the same problem or discussion of the problem altogether.

**Comment by dionysous** [comment]
So having a label next to the icon in the admin bar as optional setting. Sounds good. Then there is the question, whether the label is activated by default for new installations or not.

Some people have many icons/lots of text in the admin bar, others not. There are power users and normal user. We have to consider every ascpect. 🤔 

A good solution would be to introduce this new option, which would be activated for new installations (label shown) and deactivated for "old" users / updated plugins (label hidden).
